packages/status-js/src/utils/is-clock-valid.ts

```diff
@@ -0,0 +1,16 @@
+const MAX_OFFSET = BigInt(120 * 1000)
+
+export function isClockValid(
+  messageClock: bigint,
+  messageTimestamp: Date
+): boolean {
+  if (messageClock <= 0) {
+    return false
+  }
+
+  if (messageClock > BigInt(messageTimestamp.getTime()) + MAX_OFFSET) {
```

Comment:
Covers only `>... + MAX_OFFSET` case. The `< - MAX_OFFSET` case I understand to be not handled yet (see https://discord.com/channels/624205794384281629/626416839680983072/683211306370990098).